src/alert/alert.scss

```diff
@@ -15,28 +15,29 @@
     color: t(iui-text-color);
   }
 
-  > .iui-icon {
+  &-icon {
     @include iui-icons-default;
     flex-shrink: 0;
     margin-left: $iui-m;
   }
 
-  > .iui-message {
+  &-message {
     margin: $iui-baseline $iui-m;
+  }
 
-    > a {
-      margin-left: $iui-s;
-      text-decoration: underline;
-      user-select: none;
-      white-space: nowrap;
+  &-link {
+    cursor: pointer;
+    margin-left: $iui-s;
+    text-decoration: underline;
+    user-select: none;
+    white-space: nowrap;
 
-      &:hover {
-        text-decoration: none;
-      }
+    &:hover {
+      text-decoration: none;
     }
   }
 
-  > .iui-button {
+  .iui-button {
```

Comment:
So what about this then? Maybe also `alert-close-button` or something?

Reply:
we are adding margin. Isnt that same situation as with anchor? Just difference that its our iui-button class versus anchor element in there...

Reply:
For anchor we are actually styling it by setting `color`, `text-decoration`, `cursor` and some other stuff. For button we are just positioning it using margin. But I can change the class name if you really want.

Reply:
ah ok, added back `>` in that case.

If I change to a class name, then it will have a specificity of 0-1-0 (same as `.iui-button` which resets margin) so it would have to rely on order.